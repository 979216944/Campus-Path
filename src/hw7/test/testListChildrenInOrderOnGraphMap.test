# Test ListChildren in order without multiple edges

# Load graph
LoadGraph map map.tsv

# ListChildren
ListChildren map location1
ListChildren map location2
ListChildren map location3
ListChildren map location4
ListChildren map location5
ListChildren map location6
ListChildren map location7