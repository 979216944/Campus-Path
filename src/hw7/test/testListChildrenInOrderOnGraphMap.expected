# Test ListChildren in order without multiple edges

# Load graph
loaded graph map

# ListChildren
the children of location1 in map are: location2(1.000)
the children of location2 in map are: location1(1.000) location3(1.000)
the children of location3 in map are: location2(1.000) location4(1.000)
the children of location4 in map are: location3(1.000) location5(1.000)
the children of location5 in map are: location4(1.000) location6(1.000)
the children of location6 in map are: location5(1.000) location7(1.000)
the children of location7 in map are: location6(1.000)